lib/Connection.js

```diff
@@ -293,13 +293,13 @@ Connection.statuses = {
 }
 
 Connection.roles = {
+  CLUSTER_MANAGER: 'cluster_manager',
   MASTER: 'master',
   DATA: 'data',
   INGEST: 'ingest'
 }
 
 const defaultRoles = {
-  [Connection.roles.MASTER]: true,
```

Comment:
Why don't we want to replace this defaultRole with `CLUSTER_MANAGER`?

Reply:
Since we want both `MASTER` and `CLUSTER_MANAGER` acceptable for backward compatibility, I made both of them optional and I removed the `[Connection.roles.MASTER]: true` in the defaultRoles. Currently, only `DATA` and `INGEST` are the default roles.